WindowsServerDocs/remote/remote-access/ras/multisite/configure/Step-2-Configure-the-Multisite-Infrastructure.md

```diff
@@ -235,7 +235,7 @@ To load or modify the Remote Access configuration, you can transfer the PDC emul
   
 #### <a name="ChangeDC"></a>To change the domain controller that manages server GPOs  
   
--   Run the Windows PowerShell cmdlet  `HYPERLINK "https://technet.microsoft.com/library/hh918412.aspx" Set-DAEntryPointDC` on the Remote Access server and specify the unreachable domain controller name for the *ExistingDC* parameter. This command modifies the domain controller association for the server GPOs of the entry points that are currently managed by that domain controller.  
+-   Run the Windows PowerShell cmdlet  [Set-DAEntryPointDC](https://docs.microsoft.com/en-us/powershell/module/remoteaccess/set-daentrypointdc) on the Remote Access server and specify the unreachable domain controller name for the *ExistingDC* parameter. This command modifies the domain controller association for the server GPOs of the entry points that are currently managed by that domain controller.  
```

Comment:
Language agnostic links are preferred, as long as removing the language part of the link does not break it.
```suggestion
-   Run the Windows PowerShell cmdlet  [Set-DAEntryPointDC](https://docs.microsoft.com/powershell/module/remoteaccess/set-daentrypointdc) on the Remote Access server and specify the unreachable domain controller name for the *ExistingDC* parameter. This command modifies the domain controller association for the server GPOs of the entry points that are currently managed by that domain controller.
```
(also removed redundant whitespace at EOL)